### PR TITLE
Bump NU to 1.2-preview / latest staging

### PR DESCRIPTION
**Pull request by lciolecki**

### CODE REVIEW

**Comment by avan1235** [comment]
@lciolecki thanks for your update 👌🏻 Tests are passed, there is extra newline `LoggingSinkFunction.scala` that is detected as lint error. Fix that and we could merge it as changes looks good.
btw if you need it published on GH packages with these version, then we can think of it, but we planned to publish new version with the same version of nussknacker stable versions (so e.g. when 1.2 becomes stable)

**Comment by lciolecki** [comment]
If you could be so kind and publish new version after merge it, I will be thankful. We want use nu-prinz with latest nu version from staging.

**Comment by avan1235** [comment]
Ok, then @lciolecki let's update all places in project where you find [1.0.0-SNAPSHOT](https://github.com/prinz-nussknacker/prinz/search?q=1.0.0-SNAPSHOT) to 1.2.0-STAGING and we would be ready to go if no places are missed 😉 But please add this to your PR for simplicity

File: build.sbt [hunk 1]
```diff
@@ -7,11 +7,11 @@ val repositoryName = "prinz"
 
 // Dependency versions
 val scalaV = "2.12.10"
-val nussknackerV = "1.0.0"
+val nussknackerV = "1.2.0-staging-2021-12-09-5983-f15ac11992e3a82d32f651ef4e05b5d9458f46aa-SNAPSHOT"
```
Comment (avan1235) [inline build.sbt:10]: modify also the prinz version from 1.0.0-SNAPSHOT to 1.2.0-STAGING in all places in project

**Review by avan1235 — CHANGES_REQUESTED** [review]

**Review by avan1235 — APPROVED** [review]

**Pull request merged**